auth0_flutter_platform_interface/lib/src/web/cache_mode.dart

```diff
@@ -0,0 +1,17 @@
+enum CacheMode {
```

Comment:
Should this type have API docs, since it's going to be exposed to (and used by) developers?

Reply:
Totally! I had actually done them but didn't save the file, so they didn't get included in the commit 🤦🏻 apologies!

Updated in b8e52e4